packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -499,8 +518,9 @@ export const DEFAULT_INTERMEDIATE_TOKENS_BY_CHAIN_ID = valueByChainId<string[]>(
             POLYGON_TOKENS.USDT,
             POLYGON_TOKENS.WBTC,
             POLYGON_TOKENS.QUICK,
-            POLYGON_TOKENS.TITAN,
-            POLYGON_TOKENS.IRON,
+            POLYGON_TOKENS.DFYN,
+            POLYGON_TOKENS.BANANA,
+            POLYGON_TOKENS.WEXPOLY
```

Comment:
oh man these are getting out of control, need to figure out how to be smarter with these in the future

Reply:
how bad do you think it's impacting performance right now? most of these Uni forks add their own token for a few pairs and have decent liquidity, so it makes sense to hop through.

Reply:
:shrug: Maybe not too bad on poly since there's not as many sources as mainnet.